scipy/stats/tests/test_fit.py

```diff
@@ -269,7 +269,7 @@ def cases_test_fit_mse():
                       'kstwo',  # very slow (~25 min) but passes
                       'geninvgauss',  # quite slow (~4 minutes) but passes
                       'gausshyper', 'genhyperbolic',  # integration warnings
-                      'argus',  # close, but doesn't meet tolerance
+                      'argus', 'tukeylambda',  # close, but doesn't meet tolerance
```

Comment:
```suggestion
                      'tukeylambda',  # close, but doesn't meet tolerance
```
`argus` seems to pass now.